playwright_tests/E2E/login-and-log-out-with-valid user.test.ts

```diff
@@ -0,0 +1,16 @@
+import { test, expect } from '@playwright/test';
+import config from "../config"
+const testConfig = require('../../test_codecept/e2e/config/appTestConfig');
+
+test('login and log out from EUI with valid user', async ({ page }) => {
+  const matchingUsers = testConfig.users[testConfig.testEnv].filter((user) => user.userIdentifier === 'PROD_LIKE');
+  await page.goto(config.CaseAPIBaseURL);
```

Comment:
Actually I really want you to change it now that it's become an attribute. Let's call it AppBaseURL

Reply:
Done